src/EventStore.Core/ClusterVNode.cs

```diff
@@ -526,6 +532,26 @@ public class ClusterVNode :
 					Db.Config.WriterCheckpoint.AsReadOnly(),
 					optimizeReadSideCache: Db.Config.OptimizeReadSideCache));
 
+			var streamInfoCacheCapacity = options.Cluster.StreamInfoCacheCapacity;
+			//qq thoughts
+			// 1. maybe now the default should be 0 (or even -1 to indicate dynamic or something)
```

Comment:
Set the default to 0 to mean dynamic. If it is set to anything else, override the behaviour